### PR TITLE
Add support for H3ULCB KF ES3.0 8GB machine

### PR DESCRIPTION
**Pull request by viktor-mitin**
Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Signed-off-by: Viktor Mitin <viktor_mitin@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb [hunk 1]
```diff
@@ -19,6 +19,7 @@ DOMA_CONFIG = "doma-salvator-generic.cfg"
 DOMA_CONFIG_salvator-x-h3-4x2g-xt = "doma-salvator-x-h3-4x2g.cfg"
 DOMA_CONFIG_salvator-xs-h3-4x2g-xt = "doma-salvator-x-h3-4x2g.cfg"
 DOMA_CONFIG_h3ulcb-4x2g-xt = "doma-salvator-x-h3-4x2g.cfg"
+DOMA_CONFIG_h3ulcb-4x2g-kf-xt = "doma-salvator-x-h3-4x2g-kf.cfg"
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb:23]: doma-salvator-x-h3-4x2g.cfg ?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb [hunk 2]
```diff
@@ -26,6 +26,7 @@ DOMU_CONFIG_salvator-xs-h3-xt = "domu-salvator-x-h3.cfg"
 DOMU_CONFIG_salvator-xs-h3-4x2g-xt = "domu-salvator-x-h3-4x2g.cfg"
 DOMU_CONFIG_salvator-xs-h3-2x2g-xt = "domu-salvator-xs-h3-2x2g.cfg"
 DOMU_CONFIG_h3ulcb-4x2g-xt = "domu-salvator-x-h3-4x2g.cfg"
+DOMU_CONFIG_h3ulcb-4x2g-kf-xt = "domu-salvator-x-h3-4x2g-kf.cfg"
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb]: domu-salvator-x-h3-4x2g.cfg ?

**Comment by otyshchenko1** [comment]
Why commit subject starts with "dom0: " if your patch effectively touches both recipes-dom0/recipes-domd?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb (hunk 2 above)
Comment (viktor-mitin) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/domu.bb]: fixed

**Pull request closed without merging**